app/src/js/widgets/fields/fieldRepeater.js

```diff
@@ -231,4 +252,4 @@
             }
         }
     });
-})(jQuery, Bolt);
+})(jQuery, Bolt, typeof CKEDITOR !== 'undefined' ? CKEDITOR : undefined);
```

Comment:
Call my dumb, but isn't that _literally_ the same as just doing this?

    })(jQuery, Bolt, CKEDITOR);